test/hotspot/jtreg/ProblemList.txt

```diff
@@ -79,6 +79,8 @@ compiler/codecache/CheckLargePages.java 8319795 linux-x64
 compiler/floatingpoint/TestSubnormalFloat.java 8317810 generic-i586
 compiler/floatingpoint/TestSubnormalDouble.java 8317810 generic-i586
 
+compiler/c2/cr7200264/TestIntVect.java 8325277 linux-aarch64
```

Comment:
Is this a known issue for aarch64, right?

Reply:
Yes, here is the reported bug: https://bugs.openjdk.org/browse/JDK-8325277. It's still unresolved.